lib/logstash/outputs/influxdb.rb

```diff
@@ -69,6 +72,19 @@ class LogStash::Outputs::InfluxDB < LogStash::Outputs::Base
   #
   config :coerce_values, :validate => :hash, :default => {}
 
+  # Automatically use fields from the event as the data points sent to Influxdb
+  config :use_event_fields_for_data_points, :validate => :boolean, :default => false
```

Comment:
Probably, though it's not much more than that… Instead of having to explicitly set a list of data points (`config :data_points`…), with this set to true it will use the data from the event itself (and you can combine it with `exclude_fields` to remove common fields you don't want going to Influxdb (like @timestamp and @version, for example).